### PR TITLE
add numaresourcesscheduler API

### PR DESCRIPTION
**Pull request by fromanirh**
Sketch out the numaresourcesscheduler API and bootstrap empty controller

We want to offer a simple alternative to the [secondary scheduler operator](https://github.com/openshift/secondary-scheduler-operator). This controller will have the strictest subset of features needed to deploy the [noderesourcetopologies scheduler plugin](https://github.com/openshift-kni/scheduler-plugins).

This controller will offer basic functionaliy: installation and complete cleanup. There is no seamless upgrade path to the secondary scheduler operator: the cluster admin must have to deinstall the scheduler plugin using this operator and then reinstall uisng the secondary scheduler plugin.

The scheduler plugin is stateless and quick to setup, so this operation is expected to cause minimal to none downtime.

### CODE REVIEW

File: pkg/numaresourcesscheduler/manifests/manifests.go [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+	}
+
+	if namespace != "" {
+		sa.Namespace = namespace
+	}
+	return sa, nil
+}
+
+func ClusterRole() (*rbacv1.ClusterRole, error) {
+	obj, err := loadObject(filepath.Join("yaml", "clusterrole.yaml"))
```
Comment (Tal-or) [inline pkg/numaresourcesscheduler/manifests/manifests.go:52]: ```clusterrole.nrt.yaml```

Comment (fromanirh) [reply pkg/numaresourcesscheduler/manifests/manifests.go]: fixed

**Pull request merged**